requirements-rocm.txt

```diff
@@ -8,4 +8,5 @@ botocore
 ray >= 2.10.0
 peft
 pytest-asyncio
-tensorizer>=2.9.0
\ No newline at end of file
+tensorizer>=2.9.0
+prettytable 
```

Comment:
What's prettyable used for?

Reply:
it's used here
https://github.com/ROCm/rocmProfileData/blob/nvtx_enabled/rpd_tracer/rpdTracerControl.py

We should more than likely have this in a requirements.txt from rocmProfileData then